wsd/DocumentBroker.cpp

```diff
@@ -1437,11 +1449,24 @@ void DocumentBroker::uploadAsToStorage(const std::string& sessionId,
 {
     assertCorrectThread();
 
-    uploadToStorageInternal(sessionId, uploadAsPath, uploadAsFilename, isRename, /*force=*/false);
+    const auto it = _sessions.find(sessionId);
```

Comment:
Moved up from `uploadToStorageInternal` until `uploadAsToStorage` also gets the ClientSession instance (in a follow up).